autometa/common/kmers.py

```diff
@@ -225,7 +227,7 @@ def seq_counter(assembly, ref_kmers, verbose=True):
     Returns
     -------
     dict
-        {contig:[count,count,...]} count index is respective to ref_kmers.keys()
+        {contig:[count,count,...]} count index is respective to ref_kmers keys()
```

Comment:
This was showing the `dict.keys()` method, `ref_kmers.keys()`

```suggestion
        {contig:[count,count,...]} count index is respective to ref_kmers.keys()
```